docs/data/toolpad/core/components/page-container/ActionsPageContainer.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
```

Comment:
This demo would read better if `Content` was moved to a separate file and imported so that the main tab could focus on the custom toolbar, when expanded